lib/espree.js

```diff
@@ -3,9 +3,23 @@ import TokenTranslator from "./token-translator.js";
 import { normalizeOptions } from "./options.js";
 
 
+/**
+ * @typedef {import("acorn")} acorn
+ */
+
 const STATE = Symbol("espree's internal state");
 const ESPRIMA_FINISH_NODE = Symbol("espree's esprimaFinishNode");
 
+/**
+ * @typedef {Object} EsprimaComment
+ * @property {"Block"|"Line"} type Type of the comment, can either be "Block" (multiline) or "Line" (single line).
+ * @property {string} text Contents of the comment.
+ * @property {number|undefined} start Start column of a comment.
+ * @property {number|undefined} end End column of the comment.
+ * @property {[number, number]|undefined} range The [start, end] range of a comment.
```

Comment:
Since [number,number] is also used in EsprimaToken, can we move this into its own typedef for clarity?